operating/respond-to-violations.adoc

```diff
@@ -31,8 +31,9 @@ include::modules/violation-view-policy-tab.adoc[leveloffset=+2]
 
 include::modules/policy-enforcement-deploy.adoc[leveloffset=+3]
 
+include::modules/network-policies-tab.adoc[leveloffset=+2]
+
 [role="_additional-resources"]
 .Additional resources
```

Comment:
Is this meant to look like it is attached to the "Network policies tab" section? It seems like it is associated more with the "Policy tab" section that was previously the final section. If so, move this `.Additional resources` section up above line 34.

Alternatively, if this list is meant to apply to the entire assembly and not just a specific subsection, then it should be changed to a `== Additional resource` section (per [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections)) so that it will appear in the right-nav TOC.